common/djangoapps/tedix_ro/forms.py

```diff
@@ -11,8 +11,6 @@
 from tedix_ro.models import City, School, StudentProfile, CLASSROOM_CHOICES, InstructorProfile, ParentProfile
 
 ROLE_CHOICES = (
-    ('', ''),
```

Comment:
Is it code appropriate to the current ticket?

Reply:
no, it was a bug - at the Registration page under select Role we have two empty options, one from our custom form and another one from platform here - https://github.com/raccoongang/edx-platform/blob/tedix-hawthorn-stage/openedx/core/djangoapps/user_api/helpers.py#L250